docs/_js/ErrorCodesComponent.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/* global React ReactDOM errorMap:true */
+'use strict';
+
+function replaceArgs(msg, argList) {
+  let argIdx = 0;
+  return msg.replace(/%s/g, function() {
+    const arg = argList[argIdx++];
+    return arg === undefined ? '[missing argument]' : arg;
+  });
+}
+
+function segmentify(str) {
+  const urlRegex = /(https:\/\/fb\.me\/[a-z\-]+)/g;
```

Comment:
I think we'd be ok if we did go that route. Maybe v2 :).

FWIW, test your concerns on the example on the home page. It already turns `http://website.com` into a clickable link (though not with `target=_blank`) and converts markupy things to be escaped with `&lt;`. Then it renders with `dangerouslySetInnerHTML`.